src/components/header/modals/ModalNetwork.vue

```diff
@@ -30,13 +30,51 @@
                   </div>
                   <div v-if="index === endpointKey.CUSTOM">
                     <input
-                      v-if="isCustomNetwork"
+                      v-if="selNetwork === index"
                       v-model="newEndpoint"
                       type="text"
                       placeholder="IP Address / Domain"
                       class="ip-input"
                     />
                   </div>
+                  <div v-else-if="index === endpointKey.LOCAL">
+                    <div />
+                  </div>
+                  <div v-else>
+                    <div
+                      v-if="selNetwork === index && providerEndpoints[index].endpoints"
+                      class="box--endpoints"
+                    >
+                      <div>
+                        <span class="text--md">{{ $t('drawer.endpoint') }}</span>
+                      </div>
+                      <div class="column--options">
+                        <div
+                          v-for="(endpointObj, i) in providerEndpoints[index].endpoints"
+                          :key="i"
+                        >
+                          <div
+                            class="column--network-option"
+                            @click="setSelEndpoint({ endpointObj, networkIdx: index })"
```

Comment:
There should be a loading icon or some indicator when the page is still trying to connect to the network

Reply:
https://user-images.githubusercontent.com/40356749/165281080-c3f2a72d-dd9a-4e6c-8aa8-c08cb7b464c7.MOV

Alongside the lack of a loading screen, there is an issue where choosing Astar endpoint will automatically switch to Dwellir

Reply:
I've tried to add the loading animation during connecting network but I think it takes some time to debug because the website is actually reloading even though the screen is blank.

Reply:
https://github.com/AstarNetwork/astar-apps/blob/27d59c32029895c02222fcbe63449f30b192a8de/src/config/api/polkadot/connectApi.ts#L71

Reply:
> there is an issue where choosing Astar endpoint will automatically switch to Dwellir

That is because Astar endpoint is not working right now.

Reply:
Thank you for the clarification.

Reply:
@hoonsubin 
I found out why the loading animation doesn't display on the screen. The portal is connecting to the network in the [boot file](https://github.com/AstarNetwork/astar-apps/blob/feat/network-select/src/boot/api.ts). According to quasar [documentation](https://quasar.dev/quasar-cli-webpack/boot-files), boot code is to run before the root Vue app instance is instantiated (Loading animation is located in App.vue file).